src/main/java/com/endava/pages/BasePage.java

```diff
@@ -23,4 +27,25 @@ public void quit() {
 			driver.quit();
 		}
 	}
+
```

Comment:
All methods below should not belong to BasePage, but WebDriverUtils, bus should be called within Page classes